docs/modules/ROOT/pages/security_vulnerability_process.adoc

```diff
@@ -0,0 +1,68 @@
+= Security and Vulnerability Process
+
+Critical security vulnerabilities are announced by mailing lists from suppliers, security newsletters, MELANI, and many other sources.
+This security advisories are analysed and if a vulnerability affects the system directly, the update is planned as soon as possible.
+
+Urgent upgrades are assessed and planned immediately and less urgent vulnerabilities are patched weekly according to our xref:maintenance_process[maintenance process].
+
+== General
+
+Our xref:maintenance_process[maintenance process] ensures weekly system updates (there are some exceptions on customer's systems which are maintained only monthly according to special agreements).
+We manage server systems based on Ubuntu and RedHat.
+This two big players usually provide updates rapidly.
+With this regular maintenance the known vulnerabilities are eliminated withing one week.
+
+Internal security incidents are reported according our ISO27001 certified https://handbook.vshn.ch/hb/domain_ism_incidentmanagement.html[incident management process].
+
+Our https://handbook.vshn.ch/hb/role_responsibleops.html[Responsible Ops] engineers are supervising our monitoring dashboard during office hours according to our xref:monitoring_concept[monitoring concept].
+The Responsible Ops engineers are able to react quickly during office hours.
+
+Customers with 24/7 support contracts can call the on-call engineer anytime.
+Customers with SLA premium can define services which would alerting the on-call engineer in case of critical status.
```

Comment:
Do we need these lines? Isn't that a duplication what we already document in our SLA document? Duplication leads in to the issue of having outdated document sooner or later.

Reply:
Ah yes, you're right, in this section it is really a bit duplication.

Do you think we should remove it or change it to something like below?

```suggestion
Customers can report vulnerabilities to VSHN according their https://kb.vshn.ch/products/sla_en.html[Service Level Agreement].
```


Off-topic: Where is is the legal docs repo? Looks like it is not here in the kb repo and I have to link it with the complete URL.

Reply:
Yes, that makes sense.

> Where is is the legal docs repo?

https://git.vshn.net/vshn/kb.vshn.net/-/tree/master/docs/modules/ROOT/pages